src/Configuration/RectorConfigBuilder.php

```diff
@@ -457,6 +457,9 @@ public function withPhpSets(
             $this->sets[] = LevelSetList::UP_TO_PHP_84;
         }
 
+        // make use of polyfill packages in composer.json
+        $this->sets[] = SetList::PHP_POLYFILLS;
```

Comment:
I think this should be enabled manually via passed config, for example, we use `withPhpSets(php53: true)`, it doesn't make sense to have `str_starts_with()` change? That will make too much change when it on step by step migration.

I think user needs to enable it by, eg: `withPhpSets(php53: true, polyfill: true)`

 wdyt?

Reply:
I think add new method: `withPhpPolyfill()` can be the way to go

Reply:
Makes sense, thanks :+1: